Integration portal GA (#930)
amplitude/amplitude-docs: content/collections/partners/en/integration-portal.md

Here are the high-level steps to having your integration listed within the Amplitude app. 

1. **Go to the Integration Portal**: In Amplitude, navigate to the **My Settings** page and click **Integration Portal**. If you don't see this option, send an email to [integrations@amplitude.com](mailto:integrations@amplitude.com) with your [Org ID](/docs/admin/account-management/account-settings), and the Amplitude Product Development team will provide you access.

> 2. **Configure your integration**: Once you have finalized your testing, click **Submit** via the Integration Portal and it will notify the Amplitude Engineering team to review your integration. Clicking **Release internally** will make the integration available to members of your organization. 

Reviewer: ```suggestion
2. **Configure your integration**: After you finalize testing, click **Submit** from the Integration Portal. After you submit your integration, Amplitude engineers review it. Click **Release internally** to make the integration available to members of your organization. 
```
